### PR TITLE
docs: add minimum required permissions for creating federated identity credential

### LINKED ISSUE #548: What are minimum permissions for a SPN to provision WI?
**Is your feature request related to a problem? Please describe.**
We need to automate this WI creation for an AKS cluster.
What are minimum permissions for a SPN so that we can automate this process?
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
@johndohoneyjr @nicoleserafino

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**Reason for Change**:

- Adds minimum required permissions for creating/deleting/updating federated identity credential with Azure AD Apps and user-assigned managed identity. 

**Requirements**

- [ ] squashed commits
- [ ] included documentation
- [ ] added unit tests and e2e tests (if applicable).

**Issue Fixed**:

fixes #548 

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [ ] yes
- [ ] no

**Notes for Reviewers**:

### CODE REVIEW

**Comment by aramase** [comment]
cc @udayxhegde

File: docs/book/src/faq.md [hunk 1]
```diff
@@ -26,3 +26,23 @@ It takes a few seconds for the federated identity credential to be propagated af
 ## What is the Azure Workload Identity release schedule?
 
 Currently, we release on a monthly basis, targeting the last week of the month.
+
+## What permissions are required to create a federated identity credential for Azure AD Application?
+
+Required roles:
```
Comment (enj) [inline docs/book/src/faq.md:32]: As in, you need all of these roles or a single one?

Comment (aramase) [reply docs/book/src/faq.md]: Any of these would suffice. Updated it!

**Review by enj — APPROVED** [review]
@aramase can we get an ack from someone in AAD to make sure we got this correct?

**Comment by aramase** [comment]
> @aramase can we get an ack from someone in AAD to make sure we got this correct?

For sure! I've pinged @udayxhegde for review.

**Review by aramase — COMMENTED** [review]
Confirmed with @udayxhegde on teams!

**Pull request merged**